projection/core/src/main/scala/com/lightbend/lagom/internal/projection/ProjectionRegistry.scala

```diff
@@ -29,11 +29,12 @@ import scala.concurrent.duration._
 
 @ApiMayChange
 object ProjectionRegistry {
-
-  case class StateRequestCommand(coordinates: WorkerCoordinates, requestedStatus: Status)
+  case class WorkerRequestCommand(coordinates: WorkerCoordinates, requestedStatus: Status)
+  case class ProjectionRequestCommand(projectionName: String, requestedStatus: Status)
 }
 
 @ApiMayChange
+@InternalApi
 private[lagom] class ProjectionRegistry(system: ActorSystem) {
```

Comment:
Explicitly marking this as `InternalApi` since it is visible from Java code (via Guice) so I'm being overly explicit about `ProjectionRegistry` being internal.